dts/bindings/gpio/altr,pio.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Altera PIO node
+
+compatible: "altr,pio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    ngpios:
+      required: true
+      type: int
+      description: |
+          Width of the I/O ports.
```

Comment:
These are already specified in `gpio-controller.yaml`. Please remove them from here.